test/app_profiler/app_profiler_configuration_test.rb

```diff
@@ -50,5 +50,13 @@ class ConfigurationTest < TestCase
     ensure
       AppProfiler.profile_sampler_enabled = old_status
     end
+
+    test "otel_instrumentation_enabled raises ArgumentError if opentelemetry is not installed" do
```

Comment:
will update the description since we are no longer raising.

Reply:
Hmmmm... With the approach I'm suggesting, we can likely mock the `gem` call to raise `Gem::MissingSpecError` instead.

Reply:
Mocking does not work -- because `gem` is monkey patched here? https://github.com/rubygems/rubygems/blob/bundler-v2.5.9/bundler/lib/bundler/rubygems_integration.rb#L206. 

I can remove the test altogether and just rescue `Gem::MissingSpecError` and `Gem::LoadError` in code and leave at that.